File: ProjectNewJava/app/src/main/java/com/example/projectnewjava/MainActivity.java
package com.example.projectnewjava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        this.setTitle("Wireless Remote App");

        ImageButton ac = (ImageButton) findViewById(R.id.ac);
        ImageButton projector = (ImageButton) findViewById(R.id.projector);
        ImageButton speaker = (ImageButton) findViewById(R.id.speaker);
        ImageButton tv = (ImageButton) findViewById(R.id.tv);

        ac.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent myintent = new Intent(view.getContext(), Opt_AC.class);
                view.getContext().startActivity(myintent);
            }
        });

        projector.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent myintent = new Intent(view.getContext(), Opt_Projector.class);
                view.getContext().startActivity(myintent);
            }
        });

        speaker.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent myintent = new Intent(view.getContext(), Opt_Speaker.class);
                view.getContext().startActivity(myintent);
            }
        });

        tv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent myintent = new Intent(view.getContext(), Opt_TV.class);
                view.getContext().startActivity(myintent);
            }
        });

    }
}

File: project_trial1/app/src/main/java/com/example/project_trial1/MainActivity.kt
package com.example.project_trial1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageButton

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        this.setTitle("Wireless Remote App")

        var ac: ImageButton
        var projector: ImageButton
        var speaker: ImageButton
        var tv: ImageButton

        ac = findViewById(R.id.ac)
        projector = findViewById(R.id.projector)
        speaker = findViewById(R.id.speaker)
        tv = findViewById(R.id.tv)

        ac.setOnClickListener{
            val myintent = Intent(this, Opt_AC::class.java)
            startActivity(myintent)
        }

        projector.setOnClickListener{
            val myintent = Intent(this, Opt_Projector::class.java)
            startActivity(myintent)
        }

        speaker.setOnClickListener{
            val myintent = Intent(this, Opt_Speaker::class.java)
            startActivity(myintent)
        }

        tv.setOnClickListener{
            val myintent = Intent(this, Opt_TV::class.java)
            startActivity(myintent)
        }
    }

}
